.gitignore

```diff
@@ -8,6 +8,7 @@ npm-debug.log
 *.sublime-*
 coverage
 .DS_Store
+.mason
```

Comment:
We don't have anything that should create this anymore; it should be removed from `.npmignore`. It's probably just a stale directory on your machine.